src/main/java/com/iopipe/IOpipeExecution.java

```diff
@@ -632,6 +661,23 @@ final RemoteRequest __buildRequest()
 					gen.writeEnd();
 			}
 			
+			// Are there any labels to be added?
+			String[] labels = measurement.getLabels();
+			for (int i = 0, n = labels.length; i < n; i++)
+			{
+				// Start of labels
+				if (i == 0)
+					gen.writeStartArray("labels");
```

Comment:
Can't do this before the loop?

Reply:
It will now always emit arrays even if there is nothing added to them.